src/utils/signing.ts

```diff
@@ -57,7 +58,6 @@ function constructPhantomAgent(hash: string, isMainnet: boolean) {
 }
 
 export async function signL1Action(
```

Comment:
return type of this and other `signXXX` function will be `any` since it will return data to be signed not the signature.

can change the function names to be different than signXXX since it does not sign or return signature